<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>814</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Wind Rider Smart Farm</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/icons-motorcycle-01.png</normaloff>img/icons-motorcycle-01.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1281</width>
      <height>751</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_main">
     <attribute name="title">
      <string>MAIN</string>
     </attribute>
     <widget class="QOpenGLWidget" name="openGLWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>721</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBtn_Stop">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>정지</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_CurrTemp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>14</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>현재온도</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBtn_Start">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>시작</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CurrTemp">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_CurrTemp_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>14</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>현재습도</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_CurrTemp_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>물주기</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Humidity">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_comm">
     <attribute name="title">
      <string>COMM</string>
     </attribute>
     <widget class="QPushButton" name="btn_PortOpen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Port Open</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_PortClose">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Port Close</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>9</y>
        <width>140</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>시리얼 통신 서버</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_comPort">
      <property name="geometry">
       <rect>
        <x>247</x>
        <y>20</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>15</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>COM Port</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_BaudRate">
      <property name="geometry">
       <rect>
        <x>445</x>
        <y>20</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>128000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>15</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Baud Rate</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_DataBits">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>20</y>
        <width>76</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>15</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DataBits</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Prity">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>20</y>
        <width>76</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>EVEN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ODD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>15</y>
        <width>40</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Prity</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_StopBits">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>20</y>
        <width>76</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>15</y>
        <width>70</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Stop Bits</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>1231</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Data Send</string>
      </property>
      <widget class="QTextEdit" name="textEdit_sendData">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>1051</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_send_ascii">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ASCII</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_DataSend">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Data Send</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_send_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>10진수</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_send_16">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>16진수</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>1231</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string>Data Recieve</string>
      </property>
      <widget class="QRadioButton" name="radioButton_recieve_ascii">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ASCII</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_Clear">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_recieve_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>10진수</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_recieve_16">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>16진수</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_Data_Recieve">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>1201</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1277</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
